### ISSUE TITLE
Preferences class has been deprecated in API level 29 - how should it be replaced?

### ISSUE THREAD

**Issue by maprehensive** [opendatakit/tool-suite-X#416]

**Comment by maprehensive** [comment]
What is the proper way to use Material with preferences? If not suitable, what is the replacement appropriate for Android?

**Comment by Redeem-Grimm-Satoshi** [comment]
@maprehensive I Just resolved this issue

**Comment by maprehensive** [comment]
@Redeem-Grimm-Satoshi that's great. Can you provide some details on the fix?

**Comment by maprehensive** [comment]
@Redeem-Grimm-Satoshi I just merged changes for the PR, but cannot build. Error:

> ERROR:/Users/bengreen/Documents/odk-x/services/services_app/src/main/res/values/styles.xml:6:5-19:13: AAPT: error: resource color/navDrawerSurface (aka org.opendatakit.services:color/navDrawerSurface) not found.

Waylon suggests it might be a file not added to Android Library

**Comment by Redeem-Grimm-Satoshi** [comment]
> @Redeem-Grimm-Satoshi I just merged changes for the PR, but cannot build. Error:
> 
> > ERROR:/Users/bengreen/Documents/odk-x/services/services_app/src/main/res/values/styles.xml:6:5-19:13: AAPT: error: resource color/navDrawerSurface (aka org.opendatakit.services:color/navDrawerSurface) not found.
> 
> Waylon suggests it might be a file not added to Android Library

@maprehensive Yes it is, I sent a PR containing the changes, Once merged you'll pull and everything will work fine
